nodes.lua

```diff
@@ -124,16 +154,7 @@ minetest.register_tool("abriflame:flint", {
 			minetest.set_node(pos, {name = "abriflame:" .. namesplit[3] .. "_fire"})
 		end
 
-		itemstack:add_wear(65535 / 65)
-		return itemstack
+		return old_on_use(itemstack, user, pointed_thing)
 	end,
 })
-
-
--- fire starter tool recipe
-minetest.register_craft({
-	output = "abriflame:flint",
-	recipe = {
-		{"default:mese_crystal_fragment", "default:steel_ingot"}
-	}
-})
+minetest.register_alias("abriflame:flint", "fire:flint_and_steel")
```

Comment:
Same here.


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient